packages/react-mutation-mapper/src/util/OncoKbUtils.ts

```diff
@@ -1,19 +1,14 @@
 import _ from 'lodash';
 // TODO define models in OncoKb.ts
 import {
-    Evidence,
-    EvidenceQueryRes,
-    generateQueryVariant,
     generateQueryVariantId,
+    IndicatorQueryResp,
+    IOncoKbData,
     LEVELS,
 } from 'cbioportal-frontend-commons';
 
 import { Mutation } from '../model/Mutation';
-import {
-    IndicatorQueryResp,
-    IOncoKbData,
-    OncoKbTreatment,
-} from '../model/OncoKb';
+import { TumorType } from 'cbioportal-frontend-commons/src/api/generated/OncoKbAPI';
```

Comment:
```suggestion
import { TumorType } from 'cbioportal-frontend-commons';
```

and then combine with other `cbioportal-frontend-commons` imports above.